### PR TITLE
KEP-2133: add GA criteria

### PR DESCRIPTION
**Pull request by ruiwen-zhao**
KEP-2133: add GA graduation criteria

Signed-off-by: ruiwen-zhao <ruiwen@google.com>

### CODE REVIEW

File: keps/sig-node/2133-kubelet-credential-providers/kep.yaml [hunk 1]
```diff
@@ -27,12 +28,13 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.24"
+latest-milestone: "v1.25"
```
Comment (SergeyKanzhelev) [inline keps/sig-node/2133-kubelet-credential-providers/kep.yaml:31]: please also update the https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-node/2133.yaml

File: keps/sig-node/2133-kubelet-credential-providers/README.md [hunk 2]
```diff
@@ -351,7 +351,8 @@ can be achieved using the exec plugin.
 
 #### Beta -> GA Graduation
```
Comment (SergeyKanzhelev) [inline keps/sig-node/2133-kubelet-credential-providers/README.md:352]: please update the implementation history in the end of this doc.

File: keps/sig-node/2133-kubelet-credential-providers/README.md [hunk 3]
```diff
@@ -351,7 +351,8 @@ can be achieved using the exec plugin.
...
 
-TBD
+* existed in beta for at least 1 minor release
+* e2e test using an external implementation (e.g. [GCR](https://github.com/kubernetes/cloud-provider-gcp/tree/master/cmd/auth-provider-gcp)).
```
Comment (SergeyKanzhelev) [inline keps/sig-node/2133-kubelet-credential-providers/README.md:355]: criteria lgtm

File: keps/sig-node/2133-kubelet-credential-providers/README.md (hunk 2 above)
Comment (ruiwen-zhao) [reply keps/sig-node/2133-kubelet-credential-providers/README.md:352]: Done. PTAL

File: keps/sig-node/2133-kubelet-credential-providers/kep.yaml (hunk 1 above)
Comment (ruiwen-zhao) [reply keps/sig-node/2133-kubelet-credential-providers/kep.yaml:31]: Done, PTAL

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

**Comment by deads2k** [comment]
PRR looks good

/approve

**Pull request merged**